src/native-common/Image.tsx

```diff
@@ -96,8 +106,21 @@ export class Image extends React.Component<Types.ImageProps, {}> {
         );
     }
 
+    private _onNativeRef = (ref: RN.Image|null): void => {
+        this._nativeRef = ref;
+    }
+
     public setNativeProps(nativeProps: RN.ImageProperties) {
-        (this.refs['nativeImage'] as RN.Image).setNativeProps(nativeProps);
+        if (this._nativeRef) {
+            this._nativeRef.setNativeProps(nativeProps);
+        }
+    }
+
+    getChildContext() {
+        // Let descendant Types components know that their nearest Types ancestor is not an Types.Text.
```

Comment:
>Types [](start = 26, length = 5)

I think all occurrences of `Types` here should be `RX`

Reply:
ok

---
In reply to: [159348619](https://github.com/Microsoft/reactxp/pull/404#discussion_r159348619) [](ancestors = 159348619)